man/demi_open_pipe.md

```diff
@@ -0,0 +1,45 @@
+# `demi_open_pipe()`
+
+## Name
+
+`demi_open` - Opens an existing shared memory I/O queue.
```

Comment:
Samer here, this should probably be demi_open_pipe?

Reply:
Thanks for noting this.